docs/manuals/ZAC-gebruikershandleiding/ZAC-gebruikershandleiding.md

```diff
@@ -321,6 +328,9 @@ Met de Vrijgeven knop kan een item van de behandelaar afgehaald worden. Deze fun
 3 Vul een reden in, deze wordt als toelichting gebruikt in de historie van de zaak
 
 4 Klik op ‘Vrijgeven’ om de actie uit te voeren
+> Let op! <br>Als eenmaal 'Vrijgeven' is gekozen, is dit niet meer te onderbreken.
```

Comment:
Dito hier

Reply:
Ook hier leek het me een keuze, maar wil er best in meegaan

Reply:
Aangepast, ik zie het wel